packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FLTWFInstanceManager.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Maintains instances to intercommunicate with Dart objects.
+ *
+ * When an instance is added with an instanceId, either can be used to retrieve the other.
+ */
+@interface FLTWFInstanceManager : NSObject
+/**
+ * Add a new instance to the manager.
+ *
+ * If an instance or instanceID has already been added, it will be replaced by the new values.
+ */
+- (void)addInstance:(nonnull NSObject *)instance instanceID:(long)instanceID;
+
+/**
+ * Remove the instance with instanceID from the manager.
+ *
+ * Returns the removed instance if the manager contains the instanceId, otherwise nil.
```

Comment:
Prefer using Doxygen notation for this: `@return The removed instance if the manager contains the given identifier, otherwise nil.`

Same throughout the header.